spec/workers/recurring_contribution/cancel_subscription_worker_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
+
+RSpec.describe RecurringContribution::CancelSubscriptionWorker do
+  describe '#perform' do
+    context 'when there are expired subscriptions' do
+      before do
+        create(:subscription, :paid, expires_at: Date.current)
```

Comment:
You should also create some other subscriptions that are not expired just to make sure your code isn't called with all of them.